docs/chainbridge/setup/setup-polygon-sdk.md

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
You can remove all the contents from this page as they are duplicate.

Just keep the premining information, as a list of prerequisites.

Reply:
Removed at 5261fb94e15d56179ba90ddf76cff162ba02e03a